term/helpers.go

```diff
@@ -148,13 +148,18 @@ func processRows(rows driver.Rows) error {
 
 		for i, cell := range cells {
 			var cellS string
-			switch rowsColumnTypeName.ColumnTypeDatabaseTypeName(i) {
-			case "DECIMAL", "DECN", "NUMN":
-				cellS = cell.(*asetypes.Decimal).String()
-			case "IMAGE", "BINARY", "LONGBINARY", "VARBINARY":
-				cellS = hex.EncodeToString(cell.([]byte))
-			default:
-				cellS = fmt.Sprintf("%v", (interface{})(cell))
+			// Handle non-nil values
+			if cell != nil {
+				switch rowsColumnTypeName.ColumnTypeDatabaseTypeName(i) {
+				case "DECIMAL", "DECN", "NUMN":
+					cellS = cell.(*asetypes.Decimal).String()
+				case "IMAGE", "BINARY", "LONGBINARY", "VARBINARY":
+					cellS = hex.EncodeToString(cell.([]byte))
+				default:
+					cellS = fmt.Sprintf("%v", (interface{})(cell))
+				}
+			} else {
+				cellS = "<nil>"
```

Comment:
```suggestion
			if cell == nil {
				cellS = "<nil>"
			} else {
				switch rowsColumnTypeName.ColumnTypeDatabaseTypeName(i) {
				case "DECIMAL", "DECN", "NUMN":
					cellS = cell.(*asetypes.Decimal).String()
				case "IMAGE", "BINARY", "LONGBINARY", "VARBINARY":
					cellS = hex.EncodeToString(cell.([]byte))
				default:
					cellS = fmt.Sprintf("%v", (interface{})(cell))
				}
```

The simpler branch should come first for readability.